### PR TITLE
Updates to package setup

### PR DESCRIPTION
**Pull request by larrybradley**
This PR updates the package setup:

- Updates formatting in README and fixes install command (to be used when released on PyPI)
- Updates setup.cfg to use the README as the long description
- Adds various package metadata in setup.cfg
- Adds classifiers
- Excludes running coverage on test files
- Removes unneeded pytest-tornasync test dep
- Reformats setup.py
- Reformats the license
- Fixes the GitHub Actions cron workflow, which is currently failing

### CODE REVIEW

File: setup.py
Comment (pllim) [inline setup.py:1]: This is not gonna be fun when PyPA decides to drop support for setup.py altogether. But battle for another day.

**Review by pllim — COMMENTED** [review]
Just one small question about removing `if`. Otherwise LGTM. Thanks!

File: .github/workflows/ci_cron_weekly.yml [hunk 1]
```diff
@@ -11,19 +11,18 @@ permissions:
   contents: read
 
 jobs:
-
   ci_cron_tests:
     name: ${{ matrix.name }}
     runs-on: ${{ matrix.os }}
-    if: github.repository == 'spacetelescope/pypolyclip'
```
[the remark this answers is not in the document]
Comment (larrybradley) [reply .github/workflows/ci_cron_weekly.yml:18]: I wanted to test it on my fork. :smile:

File: setup.py
Comment (larrybradley) [reply setup.py:1]: Indeed.

As a start, I'm probably going to submit another PR to move `setup.cfg` metadata to `pyproject.toml`.

File: .github/workflows/ci_cron_weekly.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (larrybradley) [reply .github/workflows/ci_cron_weekly.yml:18]: Is there a good reason to keep it?

[the remark this answers is not in the document]
Comment (larrybradley) [reply .github/workflows/ci_cron_weekly.yml:18]: I added it back.

**Review by pllim — APPROVED** [review]
Let's merge so you can build glorious wheels.

**Pull request merged**